### PR TITLE
EDE-510 Rebase develop with edly tag 3.4.2

### PR DESCRIPTION
**Pull request by tehreem-sadat**
**Ticket Link:** https://edlyio.atlassian.net/browse/EDE-510

**Description:**
The q-verse theme needs the latest changes from Edly.

### CODE REVIEW

**Review by HamzaIbnFarooq — APPROVED** [review]

**Review by imhassantariq — APPROVED** [review]

**Comment by tasawernawaz** [comment]
@tehreem-sadat wait on this before the merge, this might not be needed if theme works as expected.

**Comment by tehreem-sadat** [comment]
Closing this PR as we don't need it now.

**Pull request closed without merging**
